### ISSUE TITLE
[Bug Report]: "Monitor" fails on Windows 10 after update to 0.6.0 (VSC-541)

### ISSUE THREAD

**Issue by LMBernardo** [espressif/vscode-esp-idf-extension#288]
**Pre Bug Report Checklist**
Before reporting any bug please make sure of these points.

- [ x ] Make sure you have searched for existing bugs and features request before you post an issue.
- [ x ] This is a bug report for the ESP-IDF Visual Studio Code extension and **not** an ESP-IDF bug report.
- [ x ] I've read the docs and found no information that could have helped solving the issue.

**Describe the bug**
Attempting to run "monitor" on 0.6.0 fails on Windows. When the command is run from the palette or button, the following output is produced:
```
PS C:\Users\Anon\Projects\esp_hid_host> $Env:Path = "
>> $Env:IDF_PATH = "
PS C:\Users\Anon\Projects\esp_hid_host> $Env:IDF_TARGET = "
>> $Env:PYTHON = "
PS C:\Users\Anon\Projects\esp_hid_host> $Env:IDF_PYTHON_ENV_PATH = "
>> cls
>> C:\Users\Anon\.espressif\python_env\idf4.2_py3.8_env\Scripts\python.exe c:\Users\Anon\Git\esp-idf\tools\idf.py -p COM3 >> monitor
```

The terminal then hangs there waiting for the final string to be terminated.
Terminating the string and then running
```
C:\Users\Anon\.espressif\python_env\idf4.2_py3.8_env\Scripts\python.exe c:\Users\Anon\Git\esp-idf\tools\idf.py -p COM3 monitor
```
in the same terminal results in a FileNotFoundError in a call stack from tools.py.

Running the `idf.py` command in a new powershell terminal, however, works as expected. Something is causing the environmental variable strings to be blank and unterminated, overwriting the environmental variables with nonsense.

Downgrading to 0.5.1 fixes the issue on the same setup with the same project.

**To Reproduce**
Open any working ESP32 project and run the "monitor" command with version 0.6.0 on Windows 10.

**Expected behavior**
The "monitor" command opens the selected serial port to monitor ESP32 output.

**Screenshots**
N/A

**Environment (please complete the following information):**

- OS Name & Version: [ Windows 10 v2004 b19041.685 ]
- VSCode Version: [ 1.52.1 ]
- ESP-IDF Version: [ 4.2 ]
- Python Version: [ 3.8 ]

**Logs**
The log is attached below, although I'm not sure it contains any relevant information.
[esp_idf_vsc_ext.log](https://github.com/espressif/vscode-esp-idf-extension/files/5794411/esp_idf_vsc_ext.log)

**Extension Configuration settings**

```
"idf.espIdfPathWin": "c:\\Users\\Anon\\Git\\esp-idf"
```

```
"idf.pythonBinPathWin": "C:\\Users\\Anon\\.espressif\\python_env\\idf4.2_py3.8_env\\Scripts\\python.exe"
```

```
"idf.openOcdConfigs": [
    
        "interface/ftdi/esp32_devkitj_v1.cfg",
        "board/esp32-wrover.cfg"
    ]
```

```
"idf.customExtraPaths": "C:\\Users\\Anon\\.espressif\\tools\\xtensa-esp32-elf\\esp-2020r3-8.4.0\\xtensa-esp32-elf\\bin;C:\\Users\\Anon\\.espressif\\tools\\xtensa-esp32s2-elf\\esp-2020r3-8.4.0\\xtensa-esp32s2-elf\\bin;C:\\Users\\Anon\\.espressif\\tools\\esp32ulp-elf\\2.28.51-esp-20191205\\esp32ulp-elf-binutils\\bin;C:\\Users\\Anon\\.espressif\\tools\\esp32s2ulp-elf\\2.28.51-esp-20191205\\esp32s2ulp-elf-binutils\\bin;C:\\Users\\Anon\\.espressif\\tools\\cmake\\3.16.4\\bin;C:\\Users\\Anon\\.espressif\\tools\\openocd-esp32\\v0.10.0-esp32-20200709\\openocd-esp32\\bin;C:\\Users\\Anon\\.espressif\\tools\\ninja\\1.10.0;C:\\Users\\Anon\\.espressif\\tools\\idf-exe\\1.0.1;C:\\Users\\Anon\\.espressif\\tools\\ccache\\3.7;C:\\Users\\Anon\\.espressif\\tools\\dfu-util\\0.9\\dfu-util-0.9-win64"
```

```
"idf.customExtraVars": "{\"OPENOCD_SCRIPTS\":\"C:\\\\Users\\\\Anon\\\\.espressif\\\\tools\\\\openocd-esp32\\\\v0.10.0-esp32-20200709/openocd-esp32/share/openocd/scripts\",\"IDF_CCACHE_ENABLE\":\"1\"}"
```

```
"idf.adapterTargetName" is "esp32"
```

**Comment by brianignacio5** [comment]
This is a duplicate of #280 which is previously fixed on master branch and it will be released on v0.6.1

**Comment by LMBernardo** [comment]
Apologies, somehow I missed that bug report. Thanks for the quick reply!
